test/index.html

```diff
@@ -0,0 +1,19 @@
+<!doctype html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>recurly-js</title>
+    <link rel="stylesheet" href="https://rawgit.com/visionmedia/mocha/master/mocha.css">
+    <script src="/duotest.js"></script>
+    <script src="https://rawgit.com/visionmedia/mocha/master/mocha.js"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/sinon.js/1.7.3/sinon-min.js"></script>
+  </head>
+  <body>
+    <div id="mocha"></div>
+    <div id="dom-testbed"></div>
+    <script>mocha.setup({ ui: 'bdd' });</script>
+    <script src="/build/recurly.js"></script>
+    <script src="/build/test.js"></script>
+    <script>duotest(mocha.run());</script>
+  </body>
+</html>
```

Comment:
`duo-test` [looks for index.html](https://github.com/duojs/test/blob/9610791b000c7acf266e248a84623bd23c348b1f/lib/index.js#L286) to run tests in a browser.